### PR TITLE
Shorten action.yml description for marketplace

### PR DESCRIPTION
**Pull request by jwpereira**
Attempting to get into the marketplace now (woooo 🥳 ). First thing they flag is our descriptions being too long (needs to be < 125 characters). So, I've just grabbed the first sentence since it's straight to the point (the magic is really all @mw-hrastega).

As for the lack of punctuation at the end: that seems to be the trend with other actions 🤔

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 name: Set up MATLAB
 description: >-
-  Install MATLAB on a Linux-based GitHub-hosted agent. Currently, this action is
-  available only for public projects and does not include transformation
-  products, such as MATLAB Coder and MATLAB Compiler.
+  Setup MATLAB on a Linux-based GitHub-hosted agent
```
Comment (mw-hrastega) [inline action.yml:5]: Install MATLAB on a GitHub-hosted runner

**Review by mw-hrastega — CHANGES_REQUESTED** [review]
Suggesting some edits

File: action.yml (hunk 1 above)
Comment (acampbel) [reply action.yml:5]: I am wary of the term install. Given "Setup" is standard GitHub nomenclature, I am favor of going with that

**Review by acampbel — APPROVED** [review]

**Review by mcafaro — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (mw-hrastega) [reply action.yml:5]: Interesting. In this case, let's use the verb form "Set up".

**Review by acampbel — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (mcafaro) [reply action.yml:5]: Should this be switched to "Set up" per Houman?

Comment (jwpereira) [reply action.yml:5]: Okay so it seems that the standard across most actions is "Setup" (as opposed to "Set up"). To stay consistent with other actions in the same UI, I'll go "Setup" for now 🤔

Comment (mw-hrastega) [reply action.yml:5]: Ok, thanks for checking this 😥

File: action.yml [hunk 2]
```diff
@@ -2,15 +2,11 @@
...
```
Comment (mw-hrastega) [inline action.yml:5]: Do we want to include Linux-based in the action description?

Comment (jwpereira) [reply action.yml:5]: I'm not sure 🤔 
Methinks if someone sees it up front, they'll be less likely to try it on a currently unsupported runner 🤔 
But I can also see the case for dropping it altogether too 🤔

Comment (mw-hrastega) [reply action.yml:5]: Oh, one more thing. GitHub-hosted "runner". Right?

File: action.yml (hunk 1 above)
Comment (mw-hrastega) [reply action.yml:5]: Hi, Johan. Using "setup" as a verb is incorrect and weird. If you like to use it, please use it as a noun. In other words, change "Setup MATLAB ..." to "MATLAB setup ...".

My preference, however, is to use a verb phrase (as in the actions listed here: https://github.com/marketplace?type=actions&query=set+up).

Comment (acampbel) [reply action.yml:5]: TL;DR - Why dont we compromise and go with the verb phrase "Set up" or "Sets up". I think that we might be in the middle of a transition where Setup as a verb might be getting more allowable in the language, given the language of tech does seem to use it more often. 

In  addition to Houman's link this one is also interesting:
https://github.com/marketplace?type=actions&query=setup

Some very notable projects use setup as a verb (Node, Go) but others are using Setup in the title but using "Set up" or "Sets up" in the description. Its all over the place, abut even if I am right and we are in a transition, I hitch Houman's is the only proposal that is correct either way.

Interesting tech style guide also supports Houman's preference
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/set-up-setup

Someday we will we learn that Houman is *always* on point.

Comment (acampbel) [reply action.yml:5]: To be clear, I think we should go with the title as Setup (this is standard for actions, and it is a title like Microsoft's Setup.exe case), but the description as Set up or Sets up

Comment (mw-hrastega) [reply action.yml:5]: Sounds good, Andy. This way we will be aligned with the marketplace community :).

**Review by mw-hrastega — APPROVED** [review]

**Review by acampbel — APPROVED** [review]

File: action.yml (hunk 2 above)
Comment (jwpereira) [reply action.yml]: Thank you! Done 😄

File: action.yml (hunk 1 above)
Comment (jwpereira) [reply action.yml]: Awesome discussion. I've gone with "Setup" in the title, and "Set up" in the description 😄

**Pull request merged**